patterns/design-patterns-2/src/main/java/com/baeldung/hexagonal/AccountApplication.java

```diff
@@ -0,0 +1,21 @@
+package com.baeldung.hexagonal;
```

Comment:
This stuff isn't part of the article. Have you got your PR coming from the wrong branch?

Reply:
Sorry, it seems I accidentally picked also my evaluation article codes into my branch which never merged but stayed in my fork. I will remove them.